store/proxy/proxy_store.go

```diff
@@ -283,10 +283,11 @@ func getNamespace(apiContext *types.APIContext, opt *types.QueryOptions) string
 	}
 
 	for _, condition := range opt.Conditions {
-		if condition.Field == "namespaceId" && condition.Value != "" {
+		mod := condition.ToCondition().Modifier
+		if condition.Field == "namespaceId" && condition.Value != "" && mod != types.ModifierIn && mod != types.ModifierNotIn {
```

Comment:
Would this also works with `condition.Field == "namespaceId" && condition.Value != "" && mod == types.ModifierEq`.  It seems like not modifier really valid here except "equals".

Reply:
I think you are right, should check if the modifier is ModifierEQ